webapp/components/Form.tsx

```diff
@@ -65,6 +110,26 @@ const Form = (): JSX.Element => {
       >
         Refusjonsskjema
       </Typography>
+      <Input 
+      name="Telefonnummer"
+      value={phoneNumber}
+      updateForm={setPhoneNumber}
+      helperText="Ditt telefonnummer med landskode" />
+      <Input 
+      name="Passord"
+      value={password}
+      updateForm={setPassword}
+      helperText="Ditt passord på medlem.ntnui.no"
+      type="password"/>
+      <Button variant="contained"
+        color="primary"
```

Comment:
Trivial tip: Since we use mui and we have the icon-pack, could you perhaps add an icon on the button? E.g. "FileDownload" 🤩

```suggestion
        color="primary"
        startIcon={<FileDownload/>}
```